PT-2109 - pt-mysql-summary throws an error when using sql_mode="ANSI_QUOTES"

svetasmirnova opened the pull request:
…QUOTES"

- Fixed issues still visible with SQL mode ANSI_QUOTES
- Added test cases for this mode

- [ ] The contributed code is licensed under GPL v2.0
- [ ] Contributor Licence Agreement (CLA) is signed
- [ ] util/update-modules has been ran
- [ ] Documentation updated
- [ ] Test suite update

svetasmirnova committed:
PT-2109 - pt-mysql-summary throws an error when using sql_mode="ANSI_QUOTES"

- Fixed issues still visible with SQL mode ANSI_QUOTES
- Added test cases for this mode

saikumar-vs commented on t/pt-mysql-summary/pt-2109.t:
```diff
@@ -0,0 +1,80 @@
+    }
+}
+
+if (!$has_keyring_plugin) {
+    plan skip_all => 'Keyring plugins are not enabled.';
```
Do we really need to skip this test if "Keyring" plugin is not enabled?

svetasmirnova replied:
Good point! Removed.

svetasmirnova committed:
PT-2109 - pt-mysql-summary throws an error when using sql_mode="ANSI_QUOTES"

- Removed keyring plugin check in the beginning of the test, because
  reported error would show up even on servers that do not have keyring
  plugin installed. Same applies to wsrep_on check and Percona Server
  features checks.
- Removed temporary dir setup, because not needed for this test.
